src/app/api/routes/ping.py

```diff
@@ -5,6 +5,4 @@
 
 @router.get("/ping")
```

Comment:
Shouldn't we remove the ping route altogether ? If so, I don't think it is necessary to open another PR just for that.

Reply:
Agreed!